style.css

```diff
@@ -116,3 +116,365 @@
   gap: 19px;
   align-items: baseline;
 }
+
+/* #work-container {
+  display: grid;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 12px;
+  background-color: #fff;
+} */
+
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** Kindly make sure to delete all the commented code lines to make your code base cleaner and professional.

Reply:
@tresorsawasawa Alright! I will make sure of that. Thank you🙏